iot/api-client/beta-features/manager/src/test/java/com/example/cloud/iot/examples/GatewayIT.java

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright 2018 Google Inc.
```

Comment:
/FYI Some of the samples have LLC instead of Inc (which is correct)  but some of the sample repos have checks that enforce Inc. For now, leave as-is, but just remember for the future we may need to update these.